src/list/components/item-fields.js

```diff
@@ -166,7 +174,8 @@ export class EditItemFields extends React.Component {
           <Localized id="item-fields-password">
             <LabelText>pASSWORd</LabelText>
           </Localized>
-          <PasswordInput className={styles.password}
+          <PasswordInput className={styles.password} required
+                         errorEl={passwordErrorEl}
```

Comment:
I'm curious why this approach was necessary, which is different from the approach taken for the hostname \<Input />.

Reply:
This is because the errorElement needs to be a sibling of the input, and passwordInput does some wrapping of the element